src/tests/audition_tests/utils.py

```diff
@@ -6,22 +6,26 @@
     ModelGroupFactory,
     init_engine,
     session,
+    MatrixFactory
 )
 
-
 def create_sample_distance_table(engine):
     ensure_db(engine)
     init_engine(engine)
     model_groups = {
         'stable': ModelGroupFactory(model_type='myStableClassifier'),
         'spiky': ModelGroupFactory(model_type='mySpikeClassifier'),
     }
+    # Creates an entry in the matrix db with uuid = "efgh" for all the models to referenece
+    DefaultMatrix = MatrixFactory()
```

Comment:
Sorry for not explaining my comment better. Since you added this line to the model factory:

https://github.com/dssg/triage/pull/397/files#diff-d1623fa8b438e100388d922ba37b4ad1R67

You don't need to do anything with it in these test files *unless you want to override the default matrix factory*; for instance, if you wanted to change one of the default attributes.

Reply:
I tried that initially, but then every time it created a new model, it tried to insert a new matrix with the same uuid into the db, but that is enforced to be unique, so it failed. I found that if I initialized a matrix once and fed it to ModelFactory, it didn't try to insert it repeatedly.

Reply:
Were you using session.add? I think you can use session.merge instead